internal/template-validator/validator/app.go

```diff
@@ -5,6 +5,7 @@ import (
 	"net/http"
 
 	templatev1 "github.com/openshift/api/template/v1"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
This dependency is not listed in `go.mod` file and not downloaded in `vendor` directory. It will be added automatically if you run:
```
go mod tidy
go mod vendor
```